Kernel/Net/NetworkAdapter.cpp

```diff
@@ -76,7 +76,7 @@ void NetworkAdapter::did_receive(ReadonlyBytes payload)
     m_bytes_in += payload.size();
 
     if (m_packet_queue_size == max_packet_buffers) {
-        // FIXME: Keep track of the number of dropped packets
+        m_packets_dropped++; 
```

Comment:
Hi, you've got a trailing space at the end of this line - that's why the CI is unhappy. Please modify this commit to fix that. (If you're not comfortable with how to do that, feel free to ask here or on Discord.)

Reply:
Just amended the changes, that was my first time using amend so please let me know if it didn't turn out correctly.

Reply:
Looks good to me! I'm not familiar with this part of the code so I'll let someone else give it a check though.